docs/devsite-help/troubleshooting.md

```diff
@@ -218,3 +218,62 @@ native libraries are copied appropriately.
 
 This information is now in the [protocol buffers guide](protobuf.md#repeated-fields-and-maps); this section has been
 retained for the sake of existing links.
+
+## How can I get ErrorInfo from RpcException?
+
+`ErrorInfo` in `RpcException` provides a consistent and human readable error details.
+It contains three major fields - reason, domain and metadata described as below:
+
+- `Reason` - It is a constant value and indicates the proximate cause of 
+the error. Error reasons are unique within a particular domain of errors.
+
+- `Domain` - It is the logical grouping to which the error belongs. It is 
+typically the registered service name of the tool or product that 
+generates the error. Example: "pubsub.googleapis.com". If the error 
+is generated by some common infrastructure, the error domain must be 
+a globally unique value that identifies the infrastructure. 
+For Google API infrastructure, the error domain is "googleapis.com".
+
+- `Metadata`- It is the additional structured details about the error. 
+It can help in identifying the exact cause of the error.
+
+```csharp
+using Google.Api.Gax.Grpc;
+try
+{
+  // Code that can throw the RpcException.
+}
+catch (RpcException ex)
+{
+   var errorInfo = ex.GetErrorInfo();   
+   var domain = errorInfo.Domain;
+   var reason = errorInfo.Reason;
+   var metadata = errorInfo.Metadata;
```

Comment:
```suggestion
    if (ex.GetErrorInfo() is ErrorInfo errorInfo)
    {
        var domain = errorInfo.Domain;
        var reason = errorInfo.Reason;
        var metadata = errorInfo.Metadata;
        // ...
    }
```

(It's entirely possible for `ex.GetErrorInfo()` to return null.

Reply:
We may want to have a comment about what the user can still do if there isn't an ErrorInfo, mind you. (They still have the status and the message.)

Reply:
Sure.